model/infinispan/src/main/resources/default-configs/default-keycloak-jgroups-udp.xml

```diff
@@ -44,18 +43,17 @@
              xmit_table_msgs_per_row="1024"
              xmit_table_max_compaction_time="30000"
    />
-   <pbcast.STABLE stability_delay="500"
-                  desired_avg_gossip="5000"
+   <pbcast.STABLE desired_avg_gossip="5000"
                   max_bytes="1M"
    />
    <pbcast.GMS print_local_addr="false"
-               join_timeout="${jgroups.join_timeout:500}"
+               join_timeout="${jgroups.join_timeout:2000}"
```

Comment:
@hmlnarik Do you know why default `jgroups.join_timeout` was set to 500 and default in Infinispan [default config was 2000](https://github.com/infinispan/infinispan/blob/11.0.9.Final/core/src/main/resources/default-configs/default-jgroups-udp.xml#L52)? Was it no purpose or we want to use default ISPN value?

Reply:
TL;DR: It was on purpose: The lower the time, the faster the startup.

The join timeout determines time to wait for nodes to respond to the discovery request. In the usual settings, it is run on a single node / virtual network with virtually zero network latency so the discovery finishes in almost no time.

Reply:
I see. @martin-kanis Can you please change it to 500 then?

Also, @hmlnarik Could you please look at the changes in this PR whether you can see some other option that needs to be set differently than the default? We went through the changes with @martin-kanis and it seemed all others were the default, but we might have missed something.

Reply:
LGTM as far as I can tell